### PR TITLE
Routes, models, controllers and ugly view to test

### PR DESCRIPTION
**Pull request by Steve-Jobless**
I have created routes, models, controllers and an ugly view to test if it is working. 

The controller for favourites has not been created yet.

### CODE REVIEW

File: app/controllers/venues_controller.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+class VenuesController < ApplicationController
+  def index
+    @venues = List.all
```
Comment (khatakslied) [inline app/controllers/venues_controller.rb]: Should this be `Venue.all `?

**Review by khatakslied — COMMENTED** [review]
Might not be showing up here, but in `venues_controller.rb` is there a private method for `venue_params`?

File: app/controllers/venues_controller.rb (hunk 1 above)
Comment (Steve-Jobless) [reply app/controllers/venues_controller.rb]: Lol, yeah that's right!

**Review by khatakslied — APPROVED** [review]
Looks good!

**Pull request merged**